gitlab-iscan.md

```diff
@@ -0,0 +1,175 @@
+# A Step-by_Step Guide to Scanning the Local Repo Inside your CI Using Elastio's Integrity Scan (iscan) Module
+
+**Note:** We will use GitLab in this example, but feel free to use any software version control tool you want.
+
+## 1. Set up your CI environment
+
+The first step is to set up your CI (Continuous Integration) environment. 
+
+**Note:** To run Elastio on any operating system (OS), you must provide the credentials of the AWS account holding our Elastio Stack. As a result, to secure your credentials, you must add the following key-value pairs as variables to your repo: aws_access_key_id, aws_secret_access_key, and region.
+
+## 2. Deploy Elastio inside your CI 
+
+Here is a step-by-step guide to deploying Elastio inside your CI pipeline: 
+
+### Step 1
+
+To deploy Elastio inside your CI, you need to run the following command to creat the folder that AWS reads its credentials from: 
+
+```
+mkdir ~/.aws && touch ~/.aws/config ~/.aws/credentials 
+```
+
+### Step 2 
+
+Once this command has successfully completed, the next step is to take the variables that you added to your repo and add them to the reated  `~/.aws/config ~/.aws/credentials` files as follows: 
```

Comment:
is this a typo "reated "?

Reply:
apologies - yes - it should be "created"